packages/quantic/force-app/main/default/lwc/quanticSummary/quanticSummary.js

```diff
@@ -51,18 +72,35 @@ export default class QuanticSummary extends LightningElement {
   }
 
   get range() {
-    return ` ${this.state.firstResult} - ${this.state.lastResult}`;
+    return `${this.state.firstResult}-${this.state.lastResult}`;
   }
 
   get total() {
     return this.state.total.toString();
   }
 
+  get noResultsLabel() {
+    return I18nService.format(
+        this.hasQuery ? this.labels.noResultsFor : this.labels.noResults,
+        I18nService.getTextBold(this.query));
+  }
+
   get duration() {
     if (this.state.hasDuration) {
-      return ` in ${this.state.durationInSeconds} seconds`;
+      const duration = this.state.durationInSeconds;
+      return ` ${I18nService.format(this.labels.inSeconds_plural, duration)}`;
```

Comment:
Shouldn't you call `I18nService.getLabelNameWithCount('inSeconds', duration)` instead?